basiclti/web-ifp/src/java/org/sakaiproject/portlets/SakaiIFrame.java

```diff
@@ -251,12 +251,12 @@ public void doView(RenderRequest request, RenderResponse response)
 	 * we either make a new content item from the tool or we empty the
 	 * source property.
 	 */
-	private Map<String, Object> patchContentItem(Long key, Placement placement)
+	private Map<String, Object> patchContentItem(Long contentId, Placement placement)
 	{
 		final boolean isSuperUser = SecurityService.isSuperUser();
 
 		// Look up the content item, bypassing authz checks
-		Map<String, Object> content = m_ltiService.getContentDao(key);
+		Map<String, Object> content = m_ltiService.getContentDao(contentId);
```

Comment:
This is the real fix. Other changes just to be more clear: call the var something useful instead of "key"